test/e2e-go/cli/goal/expect/tealConsensusTest.exp

```diff
@@ -54,14 +54,17 @@ if { [catch {
         "\n" { ::AlgorandGoal::Abort $expect_out(buffer) }
     }
 
-    teal "$TEST_ROOT_DIR/big-sig.teal" 2 1001 1
-    spawn goal clerk compile "$TEST_ROOT_DIR/big-sig.teal"
-    expect {
-        -re {[A-Z2-9]{58}} { ::AlgorandGoal::Abort "hash" }
-        -re {.*logicsig program size too large} { puts "bigsigcheck: pass" }
-        eof { ::AlgorandGoal::Abort $expect_out(buffer) }
-        "\n" { ::AlgorandGoal::Abort $expect_out(buffer) }
-    }
+    # we do not test anymore for a max lsig size because we let goal compile any program size,
+    # even if they will be rejected by the nodes
```

Comment:
This is fine. I had said I wanted to retain the test on old versions, but I was thinking of testing that the network doesn't accept 1001 bytes in old versions.

But I think the tests should be completely deleted, rather than remain as comment fossils for future code paleontologists.

Reply:
I agree, will delete them